ush/ufsda/yamltools.py

```diff
@@ -177,16 +177,15 @@ def calc_time_vars(config):
         config['LAND_WINDOW_LENGTH'] = config['land_window_length']
```

Comment:
Remove the following two lines (lines: 176 and 177): 
    elif 'land_window_length' in config.keys():
        config['LAND_WINDOW_LENGTH'] = config['land_window_length']